pos_customer_required_fields/README.rst

```diff
@@ -0,0 +1,121 @@
+========================================
+Point Of Sale - Customer required fields
+========================================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:5e1195e22d1cb20ca6b0ced42d6ae852966f3e8817a742d5e9c193708fd1b3ef
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpos-lightgray.png?logo=github
+    :target: https://github.com/OCA/pos/tree/16.0/pos_customer_required_fields
+    :alt: OCA/pos
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/pos-16-0/pos-16-0-pos_customer_required_fields
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/pos&target_branch=16.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module enforce users to properly define customer fields
+created from point of sale or used in pos order.
+
+From PoS config you will be able to define required fields.
+
+Using this module without `pos_customer_required` module you are
+still able to create PoS order without customer.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To define customer required field on PoS UI:
+
+#. Go to any PoS configuration *Point of Sale > Configuration > Point of Sale*.
+#. Edit Partner required fields.
+
+.. warning::
+
+    Make sure to choose only `res.partner`'s fields present on the PoS UI.
+
+Usage
+=====
+
+To create or edit partner from PoS
+
+#. Open PoS session
+#. Click on client button
+#. required filed are highlight (in green)
+
+.. figure:: https://raw.githubusercontent.com/OCA/pos/16.0/pos_customer_required_fields/static/description/pos_edit_customer.png
+
+If you choose an existing partner with missing informations
+while validating the payment you will be ask to collect missing
+customer informations.
+
+.. figure:: https://raw.githubusercontent.com/OCA/pos/16.0/pos_customer_required_fields/static/description/pos_customer_missing_fields.png
```

Comment:
This links are broken, 

Beside that I tested the module and it work as expected !

Reply:
Thanks @b0g  for the review and the feedback. I’ll fix the broken links.

Reply:
I suppose the broken link is normal until the branch is not merged!